.circleci/config.yml

```diff
@@ -83,8 +83,11 @@ jobs:
           path: ./test-results
           destination: test-results
       - run:
-          name: Code coverage
-          command: npm run coverage
+          name: Code Coverage
+          command: |
+            curl -Os https://uploader.codecov.io/latest/linux/codecov
+            chmod +x codecov
+            ./codecov -t ${CODECOV_TOKEN}
```

Comment:
I think the value of CODECOV_TOKEN needs to be set in circleci env vars, or does it read it from somewhere else?

Reply:
You can get the CODECOV_TOKEN in circleci env variables here: https://app.codecov.io/gh/LiveIntent/live-connect/settings